testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/ejb/stateful/servlet/web.xml

```diff
@@ -20,9 +20,9 @@
 ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
-<web-app xmlns="http://java.sun.com/xml/ns/javaee"
+<web-app xmlns="https://jakarta.ee/xml/ns/jakartaee"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
-         version="3.0">
+         xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_5_0.xsd"
+         version="5.0">
```

Comment:
One minor regarding these descriptors, I guess we should convert to Jakarta EE 10, in that case, we should use version 6.0, see https://jakarta.ee/xml/ns/jakartaee/#10

Reply:
The intended migration here is to EE9, not EE10.